src/Features/Core/Portable/Completion/CompletionService.cs

```diff
@@ -275,20 +275,29 @@ public virtual TextSpan GetDefaultCompletionListSpan(SourceText text, int caretP
            string filterText,
            IList<MatchResult> builder)
```

Comment:
interesting that we're using IList.  No problem, just generally surpirsed by that.